tests/e2e/vsphere.go

```diff
@@ -665,6 +665,45 @@ func (vs *vSphere) getHostUUID(ctx context.Context, hostInfo string) string {
 	return ""
 }
 
+//getAllHostsIP takes input of the HostInfo which has host uuid
```

Comment:
please update this comment, from what I can see, this method is not taking `HostInfo` input

Reply:
Corrected